lib/config/get.test.js

```diff
@@ -0,0 +1,28 @@
+const { getAppPath } = require('./get')
+
```

Comment:
Why is this test in lib/config and not under /test?

How are you supposed to run it - `npx mocha lib/config/get.test.js` gives me 
```
ReferenceError: test is not defined
    at Suite.describe (/Users/tjk/git-workspace/signalk/server-node/lib/config/get.test.js:11:3)
```

This still references getAdminPublic and getPluginConfigPublic.

Reply:
It uses `jest` instead of `mocha`. I'll probably just delete the file and move the test for the remaining functionality.